action.yml

```
@@ -9,6 +9,10 @@ inputs:
     description: 'The GITHUB_TOKEN secret'
     required: true
     default: ${{ github.token }}
+  github-api-url:
+    description: "Github API URL"
+    required: true
+    default: "https://api.github.com"
```

```suggestion
    default: ${{ github.graphql_url }}
```

How about this template address?

---

Good suggestion to use the github context. I kept using `github.api_url` for reasons explained in the other comment.